infra/sync/src/test/java/com/simprints/infra/sync/config/testtools/Models.kt

```diff
@@ -20,6 +20,7 @@ import com.simprints.infra.config.store.models.Vero1Configuration
 import com.simprints.infra.config.store.models.Vero2Configuration
 
 internal val generalConfiguration = GeneralConfiguration(
+    listOf(GeneralConfiguration.Modality.FACE),
```

Comment:
Typo, duplicate?

Reply:
I suppose you are referring to the fact that you have the same line twice!?
That's correct - one is for the modalities configuration and the other is for the matchingModalities configuration. That's what happens when we don't use parameter names.